collections/_posts/2023-03-08-Variationnal_Dropout_and_the_local_reparameterization_trick.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: review
+title: "Variational Dropout and the Local Reparameterization Trick"
+tags: Bayesian Inference, Distribution, Dropout, 
+author: "Romain Deleat-besson"
+cite:
+    authors: "Kingma, D. P., Salimans, T., & Welling, M. "
+    title:   "Variational Dropout and the Local Reparameterization Trick"
+    venue:   "Advances in neural information processing systems (NeurIPS), 2015"
+pdf: "http://www.pachecoj.com/courses/csc665-1/papers/kingma15.pdf"
+---
+
+
+
+# Notes
+
+* Link of a useful video: [video](https://www.youtube.com/watch?v=nKM9875PVtU&ab_channel=KapilSachdeva)
+* This post was made to establish the basis of variational dropout for a future post on sparse variational dropout.
+
+&nbsp;
+
+
+# Highlights
+
+* The objective of this paper is to have Bayesian posterior over the neural network **parameters**.
+* It is also introducing the *Local Reparameterization trick*
+* Finally, both previous points help to developp *variational dropout*
+
+
+&nbsp;
+
+
+# Introduction
+
+Has it is well known nowadays, deep neural networks can scale to millions of parameters. 
+However, this flexbility may leads to overfitting therefore not being robust for new data.
+Various regularization techniques are used in practice such as dropout for instance.
+
+
+In a previous paper that they cite, it was shown that binary dropout has a Gaussian approximation called Gaussian dropout.
+Based on that, they show that a relationship exists between dropout and Bayesian inference to greatly improve the efficiency of variational Bayesian inference on the **model parameters**. 
+The problem they underline is that :
+> Bayesian methods for inferring a posterior distribution over neural network weights have not yet been shown to outperform simpler methods such as dropout.
+
+
+That's why they proposed a new trick called the *Local reparameterization trick* that will improve the stochastic gradientbased variational inference with minibatches of data. 
+Leading to an optimization speed on the same level as fast dropout.
```

Comment:
This sentence should be rephrased (e.g. by making it the end of the last sentence), because it's not a proper sentence right now.